tests/test_tensorflow_keras_models.py

```diff
@@ -392,25 +384,25 @@ def test_preview(self, tmpdir, api, loss, optimizer, metrics):
         tiledb_model_obj = TensorflowKerasTileDBModel(uri=tiledb_uri, model=model)
         if model.built:
             assert type(tiledb_model_obj.preview()) == str
-        else:
```

Comment:
Why is this removed?

Reply:
Because of the fact that model preview is tried in the constructor, I had to change handling there. I changed that again, please check the latest commit.